config.cson

```diff
@@ -2701,9 +2701,12 @@ fileIcons:
 
 	Minecraft:
 		icon: "minecraft"
-		scope: "forge-config"
-		match: /^mcmod\.info$/i
-		colour: "dark-green"
+		match: [
+			[/^mcmod\.info$/i,"dark-green",{scope:"forge-config"}]
```

Comment:
Please add a space between each comma/colon and the following value.

The curly brackets around `scope:` are also unnecessary and should be removed to be consistent with the rest of the file's formatting.